mod/network.php

```diff
@@ -518,25 +499,20 @@ function networkThreadedView(App $a, $update, $parent)
 		$o .= status_editor($a, $x);
 	}
 
-	// We don't have to deal with ACLs on this page. You're looking at everything
-	// that belongs to you, hence you can see all of it. We will filter by group if
-	// desired.
+	$sql_table = $update ? '`item`' : '`thread`';
 
-	$sql_post_table = '';
-	$sql_options = ($star ? " AND `thread`.`starred` " : '');
-	$sql_extra = $sql_options;
-	$sql_extra2 = '';
-	$sql_extra3 = '';
-	$sql_table = '`thread`';
-	$sql_parent = '`iid`';
+	$sql_extra = ($star ? " AND `thread`.`starred` " : '') .
+		($conv ? " AND $sql_table.`mention`" : '') .
+		($nets ? sprintf(" AND $sql_table.`network` = '%s' ", DBA::escape($nets)) : '');
 
-	if ($update) {
-		$sql_table = '`item`';
-		$sql_parent = '`parent`';
-		$sql_post_table = " INNER JOIN `thread` ON `thread`.`iid` = `item`.`parent`";
+	if ($datequery) {
+		$sql_extra .= Strings::protectSprintf(sprintf(" AND $sql_table.received <= '%s' ",
```

Comment:
Please use `DBA::mergeConditions` instead of these `sprintf()` calls.

Reply:
No, not at this stage of the process. This here had just been some cleaning up. The switch to using `DBA` functions will be done when the item view(s) had been created. This PR here is just for testing if my simplifications are okay.

Reply:
As long as you have it in the back of your mind.

Reply:
My whole rework is based on replacing those two `q(` queries. For this the structure needs to be simplified. This step is done. Now I hope that anyone does notice any side effects. Then I can create that view and perform this whole parameter magic.

Question: Would you then like to continue by converting this into a module or should I finish it completely?

Reply:
I'm busy with CSS that I know you will not pick up if I put is aside, so I'd prefer if you did it yourself. I'll be providing feedback from the sidelines.